<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="northButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="westButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eastButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="southButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QLabel" name="roomDescription">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>room</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="roomGraphicsView">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>391</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="combatGraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="take">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>280</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playerInventory">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>player inventory</string>
    </property>
   </widget>
   <widget class="QLabel" name="HPBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>490</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HP Bar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Items">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Items</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="healthBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
